File: backend/Dtos/PizzaInOrderDto.cs
using ContosoPizza.Models;

namespace ContosoPizza.DTOs
{
    public class PizzaInOrderDto
    {
        public int Id { get; set; }
        public string Name { get; set; } = "";
        public double Price { get; set; }
        public bool IsGlutenFree { get; set; }
        public bool IsMenuPizza { get; set; }
        public List<PizzaTopping> Toppings { get; set; } = new();
        public int Quantity { get; set; } = 1; // new
    }
}

File: frontend/src/types/PizzaInOrderDto.ts
import type { PizzaTopping } from "./Topping";

export interface PizzaInOrderDto
{
    id:number;
    name:string;
    price: number;
    isGlutenFree: boolean;
    isMenuPizza:boolean;
    toppings:PizzaTopping[];
    quantity :number;
}